app/views/SubmittedView.scala.html

```diff
@@ -60,11 +62,12 @@
 
     @paragraph(id = Some("paragraph3"), content = HtmlContent(messages("submitted.paragraph3")))
 
-    @paragraph(
-        id = Some("exit-survey"),
-        content = HtmlContent(
-            link("survey-link", messages("exitSurvey.linkText"), controllers.auth.routes.AuthController.signOut).toString +
-            messages("exitSurvey.timeText")
-        )
-    )
+    <div id="exit-survey" class="govuk-body govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-m">@messages("exitSurvey.heading")</h2>
+        <p class="govuk-body">@messages("exitSurvey.p1")</p>
+        <p class="govuk-body">
+            <a class="govuk-link" href="@appConfig.exitSurveyUrl">@messages("exitSurvey.link")</a>
+            @messages("exitSurvey.p2")
+        </p>
+    </div>
```

Comment:
As this code is duplicated in several files, you could create a new twirl template for the feedback survey and pass it to each page that requires it. It would make future changes easier and reduce duplication